.changeset/cyan-seahorses-itch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/core-components': minor
```

Comment:
Just a `patch` for this one actually, since it's not a breaking change and it's at 0.x